maven-config/pom.xml

```diff
@@ -123,6 +123,16 @@
         <updatePolicy>always</updatePolicy>
       </snapshots>
     </repository>
+    <repository>
+      <id>nexus.axonivy.com</id>
+      <url>https://nexus.axonivy.com/repository/maven-snapshots/</url>
```

Comment:
fyi @ivy-cst , nor sure whether this was the best repo to use. 
I thought we have a composition repo already, where all our snapshots are served?